### ISSUE TITLE
vnext "Quick summary" section in release notes is not accessible

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode-docs#256]
https://github.com/Microsoft/vscode-docs/blob/vnext/release-notes/latest.md

The page would read:

> There is a lot to be excited about in this release, here is a quick summary:
> March summary
> Read on to get into the details...

**Comment by gregvanl** [comment]
Changed to bullet list
